backend/src/zac/contrib/dowc/views.py

```diff
@@ -57,6 +57,9 @@ def post(self, request, bronorganisatie, identificatie, purpose):
         document = self.get_object(bronorganisatie, identificatie)
         referer = request.headers.get("referer", "")
         response, status_code = create_doc(request.user, document, purpose, referer)
+        if status_code == 400:
+            return Response(response, status=status_code)
```

Comment:
This would then become:

```suggestion
        try:
            response, status_code = create_doc(request.user, document, purpose, referer)
        except DOWCCreateError as err:
            raise exceptions.ValidationError(err.args[0])
```